library/sanitize.inc.php

```diff
@@ -77,3 +77,17 @@ function isWhiteFile($file)
 
     return false;
 }
+
+// check if a the first parameter contains a non-numeric value
+function sanitizeNumber($number,$name='')
```

Comment:
Lets simplify this and remove the second parameter

Reply:
ok